qbertconfig/tests/base.py

```diff
@@ -31,18 +34,20 @@
 
 
 class QcTestCase(unittest.TestCase):
-    def setUp(self):
-        # set vars as if we were a user at the command line
+    @patch('qbertconfig.QbertConfig.QbertConfig._initialize_qbert_client')
+    def setUp(self, mock_qbert_client):
+        # Don't need a working qbert client for unit tests
+        mock_qbert_client.return_value = None
 
         # create temporary file to use for kubeconfig
         with tempfile.NamedTemporaryFile(prefix='qbertconfig', delete=False) as kcfg_f:
             self.kubeconfig_path = kcfg_f.name
 
         # load one profile into here
-        self.kubeconfig = Kubeconfig(kcfg_path=self.kubeconfig_path, kcfg=samples.BASE_TEST_KUBECONFIG)
+        # use vars as if we were a user at the command line
+        self.qc = QbertConfig(**{'kcfg_path': self.kubeconfig_path, 'kcfg': samples.BASE_TEST_KUBECONFIG})
```

Comment:
Since this is only test code, maybe we can get away with dumping the sample kcfg contents _into_ the file and instantite `QbertConfig` with only `kcfg_path`. I'm not a fan of passing the raw kcfg dictionary through `QbertConfig` just for it to get passed through..

Reply:
Yeah that's very doable, like this?
```
        with open(self.kubeconfig_path, "w+") as kcfg_f:
            kcfg_f.write(safe_dump(samples.BASE_TEST_KUBECONFIG))

        self.qc = QbertConfig(**{'kcfg_path': self.kubeconfig_path})
```

Reply:
^ that didn't work

Reply:
I tried this and was unable to get it to work as well. I think its because of the dependency between `QbertConfig` and `Kubeconfig` - I think we may be able to simplify this by keeping the `QbertConfig` functionality within `Kubeconfig`